libs/blocks/chart/chart.js

```diff
@@ -211,7 +217,7 @@ const areaSeriesOptions = (firstDataset) => (
 export const getChartOptions = (chartType, data, colors, size) => {
   const hasOverride = Object.keys(data.data[0]).some((header) => header.toLowerCase() === 'color');
   const dataset = processDataset(data.data);
-  const unit = data?.data[0]?.Unit || '';
+  const units = data?.data[0]?.Unit?.split('-') || [];
```

Comment:
Are you using the 'unit2' column? Also I realized this is not case insensitive like line L218 or L301.
```
  const unit = data?.data[0][Object.keys(data?.data[0]).find((key) => key.toLowerCase() === 'unit')] || '';
```